pkg/didcomm/packager/packager.go

```diff
@@ -73,8 +77,17 @@ func New(ctx Provider) (*Packager, error) {
 }
 
 func (bp *Packager) addPacker(pack packer.Packer) {
-	if bp.packers[pack.EncodingType()] == nil {
-		bp.packers[pack.EncodingType()] = pack
+	packerID := pack.EncodingType()
+
+	_, ok := pack.(*authcrypt.Packer)
+	if ok {
+		// anoncrypt and authcrypt have the same encoding type
+		// so authcrypt will have an appended suffix
+		packerID += authSuffix
```

Comment:
I don't see `UnpackMessage()` accounting for this suffix... it's just taking the `Typ` as-is. Is that for a followup PR?

Reply:
it's further down below at line [184](https://github.com/hyperledger/aries-framework-go/pull/2068/files/ff7c21b718eeb3fcf2e072a2c390176340e995e6#diff-68bdfb240e9da84a7ae7cdd37c85a8e4R164).

when Pack is called, we know the packer type, so we can check if it's authcrypt, but when unpack is called, we need to discover the packer to use which is done via the envelope's contentType. The comment at line 164 explains the rest.